testing/tools/integration/end_points.py

```diff
@@ -296,7 +296,17 @@ def resume(self):
         self.pause_event.clear()
 
     def send(self, bs):
-        self.sock.sendall(bs)
+        try:
+            self.sock.sendall(bs)
+        except OSError as err:
+            if err.errno == 104 or err.errno == 54:
+                # ECONNRESET on Linux or macOS, respectively
+                is_econnreset = true
```

Comment:
`true` and `false` should be `True` and `False` because Python.

Reply:
Ouch, thanks!  I'll fix that and also undo the CI-testing-only-raise-error commit 
886cedc since the ECONNRESET error seems to be rare enough that we may have to wait weeks for it to appear again.